### PR TITLE
docs(plus): add TeamCity integration docs

### PR DESCRIPTION
**Pull request by cy6erskunk**

### CODE REVIEW

File: docs/plus/10-teamcity.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+pageTitle: TeamCity
+menuTitle: TeamCity
+
+Running Karma in your [TeamCity] build is as simple as adding command line build
```
Comment (maksimr) [inline docs/plus/10-teamcity.md]: @cy6erskunk you forgot put link on TeamCity :)

Comment (cy6erskunk) [inline docs/plus/10-teamcity.md]: It is in the end of the page (kinda advanced link syntax, you know ;))

**Comment by maksimr** [comment]
**LGTM**

**Comment by Dignifiedquire** [comment]
LGTM

**Pull request merged**